Tools/machines/perlmutter-nersc/perlmutter_gpu.sbatch

```diff
@@ -15,7 +15,7 @@
 # A100 40GB (most nodes)
 #SBATCH -C gpu
 # A100 80GB (256 nodes)
-#S BATCH -C gpu&hbm80g
+#SBATCH -C gpu&hbm80g
```

Comment:
Sometimes these spaces are intentional (i.e., they are used to mark features that are not yet available). 
By making this change, do you mean that 80GB GPUs are generally available on Perlmutter and that a user should prefer them by default over the 40 GB GPU nodes?